assets/js/components/setup-module.js

```diff
@@ -106,6 +107,7 @@ class SetupModule extends Component {
 			requiredModules.forEach( ( requiredModule ) => {
 				if ( ! modules[ requiredModule ].setupComplete ) {
 					blockedByParentModule = true;
+					parentModule = requiredModule.replace( /(^\w{1})|(\s{1}\w{1})/g, ( match ) => match.toUpperCase() );
```

Comment:
We can't simply transform a module slug into a name for display like this as the module name is a translatable string. We need to use `modules[ requiredModule ].name` if it exists.

Reply:
@aaemnnosttv if the `modules[ requiredModule ].name` does not exist, is there a fallback ?

Reply:
The required module should always exist. It already assumes this above where it checks if `setupComplete` so we can assume it exists here as well.

Reply:
Fixed as requested